src/utils.js

```diff
@@ -1,68 +1,158 @@
+  newElement.innerHTML = template;
+  return newElement.firstChild;
+};
+
 /**
- * @description generate random integer
+ * generate random integer
  * @param {number} a
  * @param {number} b
  * @return {number}
  */
-export const getRandomInteger = (a = 0, b = 1)=> {
+export const getRandomInteger = (a = 0, b = 1) => {
   const lower = Math.ceil(Math.min(a, b));
   const upper = Math.floor(Math.max(a, b));
   return Math.floor(lower + Math.random() * (upper - lower + 1));
 };
 
-export const humanizeDate = (date)=> {
-  return date.toLocaleString(
-      `en-US`, {
-        day: `numeric`,
-        month: `short`
-      });
+export const capitalize = (string) => {
+  return string.charAt(0).toUpperCase() + string.slice(1);
 };
 
-// Transform date to hours and minutes
-export function prettyDate(time) {
-  return time.toLocaleTimeString(navigator.language, {
-    hour12: false,
-    hour: `2-digit`,
-    minute: `2-digit`,
-  });
-}
 
-export const getTimeBetween = (startDate, endDate) => {
-  const gap = endDate.getTime() - startDate.getTime();
+/**
+ * generate random start date between -dayGap and dayGap
+ * @return {moment.Moment}
+ */
+export const generateStartMoment = () => {
+  const maxDaysGap = 7;
+  return moment()
+    .add(getRandomInteger(-maxDaysGap, maxDaysGap), `days`)
+    .hours(getRandomInteger(0, 23))
+    .minutes(5 * getRandomInteger(0, 11));
+};
 
-  const day = Math.floor(gap / 1000 / 60 / 60 / 24);
-  const hour = Math.floor((gap / 1000 / 60 / 60) % 24);
-  const minute = Math.floor((gap / 1000 / 60) % 60);
+/**
+ * generate random event end date
+ * @param {object} startMoment
+ * @return {moment.Moment}
+ */
+export const generateEndMoment = (startMoment) => {
+  return moment(startMoment)
+    .add(getRandomInteger(1, 2), `days`)
+    .hours(getRandomInteger(0, 23))
+    .minutes(5 * getRandomInteger(0, 11));
+};
 
-  let gapString = ``;
+/**
+ * format date like '13:00'
+ * @param {object}inputMoment
+ * @return {string}
+ */
+export const getMomentTimeAsString = (inputMoment) => {
+  return moment(inputMoment).format(`HH:mm`);
+};
+
+/**
+ * format date like '04 or 11 or 15'
+ * @param {object} inputMoment
+ * @return {string}
+ */
+export const getMomentDaysAsString = (inputMoment) => {
+  return moment(inputMoment).format(`DD`);
+};
+
+/**
+ * format date like 'Sep 05'
+ * @param {object} inputMoment
+ * @return {string}
+ */
+export const getMomentMonthAsString = (inputMoment) => {
+  return moment(inputMoment).format(`MMM DD`);
+};
+
+/**
+ * Format date like '2020-09-14T11:05:56.926Z'
+ * @param {object} inputMoment
+ * @return {string}
+ */
+export const getMomentISOFormat = (inputMoment) => {
+  return moment(inputMoment).toISOString();
+};
+
+/**
+ * Format date like '20/09/14 06:55'
+ * @param {object} inputMoment
+ * @return {string}
+ */
+export const getMomentSlashedFormat = (inputMoment) => {
+  return moment(inputMoment).format(`YY/MM/DD HH:mm`);
+};
```

Comment:
It's a matter of taste, but I believe you don't need a separate function for every format. You could just have a bunch of global format string constants instead and use them with moment().format() directly